### ISSUE TITLE
Adding users via /admin/auth/user/add/ throws error because password field doesn't exist

### ISSUE THREAD

**Issue by jlaso** [ctxis/django-admin-view-permission#45]
Request Method: | GET
-- | --
http://localhost/admin/auth/user/add/
1.11
KeyError
'password'
/usr/local/lib/python3.5/site-packages/admin_view_permission/admin.py in get_form, line 339
/usr/local/bin/python
3.5.5
['/usr/src/app',  '/usr/local/bin',  '/usr/local/lib/python35.zip',  '/usr/local/lib/python3.5',  '/usr/local/lib/python3.5/plat-linux',  '/usr/local/lib/python3.5/lib-dynload',  '/usr/local/lib/python3.5/site-packages']
5 Mar 2018 17:42:33 +0000

The offending code is https://github.com/ctxis/django-admin-view-permission/blob/master/admin_view_permission/admin.py  line 339,  at least in my case the form has two password fields: password1 and password2

this is part of the rendered form

```
<div class="form-row field-password1">
 <div>
   <label class="required" for="id_password1">Password:</label>
   <input type="password" name="password1" id="id_password1" required />
   </div>
   </div>
   <div class="form-row field-password2">
   <div>
   <label class="required" for="id_password2">Password confirmation:</label>
   <input type="password" name="password2" id="id_password2" required />
   <div class="help">Enter the same password as before, for verification.</div>
   </div>
   </div>
```

**Comment by Sinkler** [comment]
@jlaso temporary you can use my branch #46 

`pip install -e git+https://github.com/Sinkler/django-admin-view-permission@exclude-models#egg=django-admin-view-permission`

**Comment by lefterisnik** [comment]
Hi all,

Thank you for the feedback.

The fix #43 has been merged to master. Tomorrow, I will push a new release.

Thanks,
Lefteris
